home.html

```diff
@@ -0,0 +1,147 @@
+                      <li class="menu-items"><a href="#" class="menu-list-a">CC Campaign</a></li>
+                    </ul>
+                    <div class="closeBtn">&times;</div>
+                  </div>
+            </nav>
+        </header>
+        <div class="homepage">
+            <h2 class="homepage-h2">"Hello! Sharing World"</h2>
+            <h1 class="homepage-h1">BEAUTY TECH AND INNOVATION SUMMIT <br> 2023</h1>
+            <p class="homepage-p">International event bringing together retail chains,distributors and beauty specialists.Discover the latest science,innovations and technologies that are revolutionizing the beauty and personal care industrywhile still retaining personalization.</p>
+            <h3 class="homepage-h3">2023.08.10(FRI) ~ 11(SAT) </h3>
+            <p class="homepage-p2">@ Bomas of Kenya Auditorium, Nairobi,Kenya.</p>
+        </div>
+    </section>
+
+    <!--Main Program-->
+    <section class="Program">
+        <h3 class="prog-h3">Main Program</h3>
+        <div class="underline"></div>
+        <ul class="prog-ul">
+            <li class="prog-li">
+                <img src="Logo/lectureicon.jpg" >
+                <h5 class="prog-h5"> Educate</h5>
+                <p class="prog-p">Learn how to identify tech solutions to your supply chain challenges and how to use tech in your formulations.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/program_icon_03 1.png">
+                <h5 class="prog-h5"> Innovate</h5>
+                <p class="prog-p">Utilize future technologies and innovations for both in-store and online retail.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/program_icon_04 1.png">
+                <h5 class="prog-h5"> Discover</h5>
+                <p class="prog-p">Discover the future of digital marketing capabilities and trends discovery using AI.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/connect.jpg">
+                <h5 class="prog-h5"> Connect</h5>
+                <p class="prog-p">Network with other professionals in the beauty industry and opportunities in the rise of holistic beauty.</p>
+            </li>
+            <li class="prog-li">
+                <img src="Logo/program_icon_02 1.png">
+                <h5 class="prog-h5"> Predict</h5>
+                <p class="prog-p">Effectively make predictions and driving consumer enagement using data.</p>
+            </li>
+         </ul>
+         <div class="button">
+            <button class="join-btn">Join Summit 2023</button>
+         </div>
+         <div class="text-btn">
+            <a href="#" class="join-text" title="See More button">SEE THE WHOLE PROGRAM</a>
+         </div>
+    </section>
+
+
+    <section class="featured-guests" id="featured-guests">
+        <h2>Featured Speakers</h2>
+        <hr class="underline">
```

Comment:
- [ ] Could you please take another look at this change? It doesn't appear as expected, and it's possible that a bug was introduced while fixing a previous suggestion. To provide a better understanding, I've attached a video below. Thank you!

   [scrnli_8_7_2023_17-50-43.webm](https://github.com/Cheptoo21/Capstone-Project-1/assets/107952724/cdfe0899-513b-49cd-8ca9-52b2110d2f82)